build.sh

```diff
@@ -1,6 +1,4 @@
 mkdir -p build/
 cd build/
 cmake ../
-cmake --build .
-
-env > env_vars.log
\ No newline at end of file
+cmake --build . -- VERBOSE=2
```

Comment:
When this command is executed, does it build the binary in debug or release mode, and what level of optimization is applied?

Reply:
I am not quite sure what would be the default option if we do not specify the build type, my guess is it would be release with 0 level of optimization. I have added  `-DCMAKE_BUILD_TYPE=Debug` to explicitly define debug build type to the Cmake.